### PR TITLE
Play a pling on player's turn

### PR DESCRIPTION
**Pull request by robinlarsson**
The feature you all been waiting for? ;P

### CODE REVIEW

**Comment by Zomis** [comment]
I haven't tested yet, but how/why will this play multiple sounds if the player has multiple actions?

Nice sound btw :D

File: games-vue-client/src/components/PlayGame.vue [hunk 1]
```diff
@@ -75,6 +77,7 @@ export default {
         this.clearActions();
         return
       }
+      this.whispered = false;
```
Comment (robinlarsson) [inline games-vue-client/src/components/PlayGame.vue:80]: The whisper resets every time the user has an action to perform. For games like Avalon when the user is having multiple actions on the same turn means that the sound will play multiple times.

Need to find a way to limit the notifications for multiple action rounds.

**Comment by robinlarsson** [comment]
> I haven't tested yet, but how/why will this play multiple sounds if the player has multiple actions?
> 
> Nice sound btw :D

Earlier the whisper reset every time the user performed an action. 

Now I reset the whisperer every time the available actions are 0.

**Comment by Zomis** [comment]
Changes incorporated by copy-pasting, I added another place to have the options at instead of the old LobbyOptions component. Many thanks!

**Pull request closed without merging**